sdk/PrebidMobileTests/PrebidMobileCoreTests/PBBidManagerTests.m

```diff
@@ -110,6 +111,20 @@ - (void)testRegisterBannerAdUnitNoSizeException {
     XCTAssertNil(returnedAdUnit);
 }
 
+- (void)testRegisterAdUnitWithSameIdentifier {
+    PBAdUnit *returnedUnit = nil;
+    PBAdUnit *bannerAdUnit1 = [[PBBannerAdUnit alloc] initWithAdUnitIdentifier:@"sameid" andConfigId:@"0b33e7ae-cf61-4003-8404-0711eea6e673"];
+    [bannerAdUnit1 addSize:CGSizeMake(320, 50)];
+    PBAdUnit *bannerAdUnit2 = [[PBBannerAdUnit alloc] initWithAdUnitIdentifier:@"sameid" andConfigId:@"0b33e7ae-cf61-4003-8404-0711eea6e673"];
+    [bannerAdUnit2 addSize:CGSizeMake(320, 50)];
+
+    [[PBBidManager sharedInstance] registerAdUnits:@[bannerAdUnit1, bannerAdUnit2] withAccountId:self.accountId];
+    returnedUnit = [[PBBidManager sharedInstance] adUnitByIdentifier:[bannerAdUnit1 identifier]];
+
+    XCTAssertNotNil(returnedUnit);
```

Comment:
Do you need to test that only one adunit with the identifier exist in the bid manager?

Reply:
Updated the test to check that the second ad unit is the one that is returned, the first ad unit gets removed.